### ISSUE TITLE
backend: editing personal information. Update of database

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#26]
## User story
1. As a {HR-employee}
2. I want / need {the information in the backend to be adjusted accordingly if I edit a digital identity via the frontend}
3. So that {all information in the system is valid}

## Acceptance criteria
* information in the backend is updated accordingly
* SDs have researched how an update of the digital identity on the blockchain is performed
* all necessary information to update the digital identity on the blockchain is stored in the backend. No necessary old information is deleted

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@jackDS008 
I hope the acceptance criteria are ok like that. Otherwise, just comment on it ;)
